lib/cldr/export/data/currencies.rb

```diff
@@ -17,8 +17,16 @@ def currencies
 
         def currency(node)
           data = select(node, 'displayName').inject({}) do |result, node|
-            count = node.attribute('count') ? node.attribute('count').value.to_sym : :one
-            result[count] = node.content unless draft?(node)
+            unless draft?(node)
+              if node.attribute('count')
+                count = node.attribute('count').value.to_sym
+                result[count] = node.content
+              else
+                result[:one] = node.content if result[:one].nil?
```

Comment:
@froyomuffin Why is `one` special cased here?

---

* This adds a `one` key for languages that don't have a `one` pluralization rule (e.g., `zh`), slightly bloating the data files)
* This is missing all of the other pluralization keys that might be needed in the locale.
  
  ```ruby
  I18n.with_locale(:af) do
    I18n.t("currencies.LVL", count: 0)
  end
  ```
  
  ```
  I18n::InvalidPluralizationData: translation data {:one=>"Lettiese lats", :name=>"Lettiese lats", :symbol=>"LVL"} can not be used with :count => 0. key 'other' is missing.
  ```

---

My guess without context is that this was done to give a "reasonable" default for the currency in cases where CLDR doesn't have pluralization translation information for the currency.

IMO, in some sense this is fabricating information that isn't in the upstream CLDR dataset.
Unless `ruby-cldr`'s mission is to augment CLDR with its own, I feel that this logic belongs downstream of `ruby-cldr` in the consumer's code.

Reply:
Ah, I see that you were just maintaining this existing behaviour (which is present all the way back to at least https://github.com/ruby-i18n/ruby-cldr/commit/666446429d46c760afc9c6b3291a24ba1668e890).

Reply:
Sorry for late reply. I honestly do not remember >.<